NSS-Day-Cohort-42/rare-media-moguls
# Created User Profile View

#### Changes Made
1. Added files `UserProfile.js` & `SubscribeButton.js` to `users` directory.
1. Modified file `UserProvider.js` and `PostList` to allow users to click the author name and be redirected to that authors profile.
1. Modified `ApplicationViews.js` to set the current user and current user profile. 

#### Steps to Review
1. Checkout this branch locally and run the application.
```
git fetch --all
git checkout mp-user-profile
```
2. Run the application.
```
npm start
```
3. Initialize virtual environment, and run the server.
```
pipenv shell
python manage.py runserver
```
4. Test app functionality.
> log in - find a post in the post list where you are the author and click the author's name - you should be redirected to a new url `/users/${userId}` to view that authors user profile with the following components;

- [ ] default image
- [ ] first and last name
- [ ] username
- [ ] date joined
- [ ] clickable posts link _(does not redirect to list of users posts yet)_
- [ ] subscribe button _(only rendered on user profiles not owned by the logged in user)_
- [ ] bio

5. View code file.
> Confirm file modifications are present as indicated above.
> Confirm no unused code or extraneous comments exist.
